modules/installing-ocp-agent-download.adoc

```diff
@@ -9,6 +9,11 @@
 
 Use this procedure to download the Agent-based Installer and the CLI needed for your installation.
 
+[NOTE]
+====
+Currently, downloading the Agent-based Installer is not supported on the {ibm-z-name} (`s390x`) architecture. The recommended method is by creating PXE assets.
```

Comment:
Just checking: Should this note have consistent wording with [this other note](https://github.com/openshift/openshift-docs/pull/69850/files#diff-712d68c5898de472eec19bbf3b4d7e1bda1c35b74926b9af5f0ac5368f3dd22cR14)?

Reply:
Different wording is on purpose

Reply:
I understand that this note and the one below are supposed to have different word, but below you have: (`s390x` architecture) with architecture in parentheses, and here architecture is not in parentheses.